Cargo.toml

```diff
@@ -15,12 +15,16 @@ rust-version = "1.63"
 exclude = ["benches/*.json", "benches/*.txt"]
 
 [dependencies]
```

Comment:
I think this might be a good time to update `rayon` to its latest version 1.10 or at least 1.8 which pulls in the latest `rayon-core` 1.12 (instead of 1.9 as done currently).

Reply:
The way Cargo interpret revision is that it will pull the highest available revision compatible according to semver.
So today, if you don't have a Cargo.lock you are already using the last rayon rev.